engine/src/main/java/org/terasology/audio/openAL/streamingSound/OpenALStreamingSoundSource.java

```diff
@@ -21,25 +21,18 @@
 import org.terasology.audio.openAL.SoundPool;
 import org.terasology.audio.openAL.SoundSource;
 
-import java.nio.IntBuffer;
+import static org.lwjgl.openal.AL10.*;
 
-import static org.lwjgl.openal.AL10.AL_BUFFER;
-import static org.lwjgl.openal.AL10.AL_BUFFERS_PROCESSED;
-import static org.lwjgl.openal.AL10.AL_PLAYING;
-import static org.lwjgl.openal.AL10.AL_SOURCE_STATE;
-import static org.lwjgl.openal.AL10.alGetSourcei;
-import static org.lwjgl.openal.AL10.alSourcePlay;
-import static org.lwjgl.openal.AL10.alSourceQueueBuffers;
-import static org.lwjgl.openal.AL10.alSourceRewind;
-import static org.lwjgl.openal.AL10.alSourceStop;
-import static org.lwjgl.openal.AL10.alSourceUnqueueBuffers;
-import static org.lwjgl.openal.AL10.alSourcei;
+import java.nio.IntBuffer;
 
-public class OpenALStreamingSoundSource extends BaseSoundSource<OpenALStreamingSound> {
+public class OpenALStreamingSoundSource
+        extends BaseSoundSource<OpenALStreamingSound> {
```

Comment:
Newline for `extends` ? The line isn't nearly long enough to worry about width :-)